dashboard/app/models/user_permission.rb

```diff
@@ -41,7 +41,9 @@ class UserPermission < ActiveRecord::Base
     # professional development workshop attendance.
     WORKSHOP_ORGANIZER = 'workshop_organizer'.freeze,
     # Grants ability to conduct peer reviews for professional learning courses
-    PLC_REVIEWER = 'plc_reviewer'.freeze
+    PLC_REVIEWER = 'plc_reviewer'.freeze,
+    # Makes the account satisfy authorized_teacher?
```

Comment:
Something that matches the pattern of the other commends would be ideal.

It looks like authorized teachers are special in that they can't have their accounts locked and are able to view teacher markdown and level examples; a brief summary of that would probably be just fine.